recipes/django-nested-inline/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.3.5" %}
+
+package:
+    name: django-nested-inline
+    version: {{ version }}
+
+source:
+    fn: django-nested-inline-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/d/django-nested-inline/django-nested-inline-{{ version }}.tar.gz
+    md5: e59b87c90a0eebcccf902ae243121578
+    patches:
+        - zipsafe.patch
```

Comment:
Has this been proposed back to the repo? It would be helpful to have an audit trail of where these came from, and when we can remove them (i.e. link to the related issue please :smile:)